src/Stripe.net/Services/ServiceNested.cs

```diff
@@ -0,0 +1,105 @@
+        {
+            return this.DeleteNestedEntityAsync(parentId, id, null, requestOptions, cancellationToken);
+        }
+
+        protected Task<EntityReturned> DeleteNestedEntityAsync(string parentId, string id, BaseOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return this.DeleteRequestAsync<EntityReturned>(this.InstanceUrl(parentId, id), options, requestOptions, cancellationToken);
+        }
+
+        protected EntityReturned GetNestedEntity(string parentId, string id, RequestOptions requestOptions = null)
+        {
+            return this.GetNestedEntity(parentId, id, null, requestOptions);
+        }
+
+        protected EntityReturned GetNestedEntity(string parentId, string id, BaseOptions options = null, RequestOptions requestOptions = null)
+        {
+            return this.GetRequest<EntityReturned>(this.InstanceUrl(parentId, id), options, requestOptions);
+        }
+
+        protected Task<EntityReturned> GetNestedEntityAsync(string parentId, string id, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return this.GetNestedEntityAsync(parentId, id, null, requestOptions, cancellationToken);
+        }
+
+        protected Task<EntityReturned> GetNestedEntityAsync(string parentId, string id, BaseOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return this.GetRequestAsync<EntityReturned>(this.InstanceUrl(parentId, id), options, requestOptions, cancellationToken);
+        }
+
+        protected StripeList<EntityReturned> ListNestedEntities(string parentId, BaseOptions options = null, RequestOptions requestOptions = null)
+        {
+            return this.GetRequest<StripeList<EntityReturned>>(this.ClassUrl(parentId), options, requestOptions);
+        }
+
+        protected Task<StripeList<EntityReturned>> ListNestedEntitiesAsync(string parentId, BaseOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return this.GetRequestAsync<StripeList<EntityReturned>>(this.ClassUrl(parentId), options, requestOptions, cancellationToken);
+        }
+
+        protected EntityReturned UpdateNestedEntity(string parentId, string id, BaseOptions options = null, RequestOptions requestOptions = null)
+        {
+            return this.PostRequest<EntityReturned>(this.InstanceUrl(parentId, id), options, requestOptions);
+        }
+
+        protected Task<EntityReturned> UpdateNestedEntityAsync(string parentId, string id, BaseOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Minor but I wonder if we should avoid the default values on parameters like requestOptions and cancellationToken. Why you ask? Because it's easy to mess up your copy/paste (we both did in this PR) and have `cancellationToken` be the default value instead of what the upper level passed.